openedx/frontend-app-course-authoring
# [FC-0036] Tags Sidebar

## Description

Adds a new tag sidebar. **Important:** This PR uses https://github.com/openedx/frontend-app-course-authoring/pull/832 as base, if you want to review the tag sidebar changes you can review [this commits](https://github.com/openedx/frontend-app-course-authoring/pull/852/files/09bf179e2828c261244cd86822e7f99426901c91..1d505dad132b3194ae3326e4b59e01b69672e531)

## Screenshots

![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/db4c28dc-3ee3-405f-a94e-48827c391f6d)

## Testing instructions

- Review the commits added by this change: https://github.com/openedx/frontend-app-course-authoring/pull/852/files/09bf179e2828c261244cd86822e7f99426901c91..1d505dad132b3194ae3326e4b59e01b69672e531
- Enable the new Unit page by adding a waffle flag `contentstore.new_studio_mfe.use_new_unit_page` in the CMS admin panel.
- Make sure that the MFE setting `ENABLE_UNIT_PAGE=true` is enabled.
- Go to the Course Unit page from the Course Outline page.
- Verify the new tags sidebar.
- Open the tags drawer and add some tags. Verify that the tags sidebar is updated in real time.
- Verify the styles of the tags sidebar.
- Open the tags drawer and delte all tags.
- Verify the opacity of tags count when is zero.

## Support information

- Closes https://github.com/openedx/modular-learning/issues/187
- Internal ticket: [FAL-3642](https://tasks.opencraft.com/browse/FAL-3642)
